src/plugins/faultManagement/FaultManagementInspector.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+<div
+    v-if="selectedFaults.length === 1"
```

Comment:
Is a for loop even necessary if there is only 1 selectedFault?
In stead of passing in a list of faults, the prop be just the 1 selectedFault from the list?

Reply:
initially started as showing all selected faults in inspector.